client/modules/workspace/src/AppsBreadcrumb/AppsBreadcrumb.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from 'react';
+import { Breadcrumb } from 'antd';
+import { Link, useParams, useLocation } from 'react-router-dom';
+import styles from './AppsBreadcrumb.module.css';
+import { TAppParamsType, TAppResponse, useGetApps } from '@client/hooks';
+
+function getPathRoutes(path: string = '') {
+  const pathComponents = decodeURIComponent(path)
+    .split('/')
+    .filter((p) => !!p);
+
+  return pathComponents.map((comp, i) => ({
+    title: comp,
+    path: '/' + encodeURIComponent(pathComponents.slice(0, i + 1).join('/')),
+  }));
+}
+
+export const AppsBreadcrumb: React.FC = () => {
+  const [appData, setAppData] = useState<TAppResponse | null>(null);
+  const { appId } = useParams() as TAppParamsType;
+  const appVersion = new URLSearchParams(useLocation().search).get(
+    'appVersion'
+  ) as string | undefined;
+
+  const { data, isLoading } = useGetApps({ appId, appVersion });
+
+  useEffect(() => {
+    if (data) {
+      setAppData(data);
+    }
+  }, [data]);
+
+  const { pathname } = useLocation();
+  const breadcrumbItems = [
+    { title: 'Home', path: window.location.origin },
+    { title: 'Use DesignSafe' },
+    { title: 'Tools & Applications', path: '/' },
+  ];
+
+  // Modify the path for Job Status
+  let modifiedPath = pathname;
+  if (pathname.endsWith('/history')) {
+    modifiedPath = 'Job Status';
+  } 
+
+  return (
+    <div className={styles.breadcrumbWrapper}>
+      {!isLoading && (
```

Comment:
We also want to show the breadcrumb when no app is loaded, i.e. at https://designsafe.dev/rw/workspace/#!/. The `isLoading` param seems to prevent the breadcrumb from immediately rendering. We can instead render the breadcrumb always, and conditionally render the final element based on the `isLoading` or `data` params